### DISCUSSION TITLE
Can this do the same thing as VSCode's @workspace directive?

### DISCUSSION THREAD

**Discussion by gisketch** [CopilotC-Nvim/CopilotChat.nvim#94] [General]
Was wondering if this can also talk to the codebase like VSC's implementation of Copilot Chat.

**Comment by jellydn** [comment]
Hi @gisketch I believe your question/suggestion is the same scope as my open issue :) https://github.com/CopilotC-Nvim/CopilotChat.nvim/issues/5 

I guess we could do it when `CopilotChat.nvim` in fully lua and more people is joining with us.
